src/CalldataList.sol

```diff
@@ -68,40 +61,33 @@ abstract contract CalldataList {
         view
         returns (Index[] memory)
     {
-        return calldataList[contractAddress][selector].calldataChecks;
+        return calldataList[contractAddress][selector];
     }
 
     /// @notice check if the calldata conforms to the expected values
     /// extracts indexes and checks that the data within the indexes
     /// matches the expected data
     /// @param contractAddress the address of the contract that the calldata check is applied to
-    /// @param value the amount of native asset sent with the call
     /// @param data the calldata to check
-    function checkCalldata(
-        address contractAddress,
-        uint256 value,
-        bytes memory data
-    ) public view {
+    function checkCalldata(address contractAddress, bytes memory data)
+        public
+        view
+    {
         bytes4 selector = data.getFunctionSignature();
 
-        Index[] storage calldataChecks =
-            calldataList[contractAddress][selector].calldataChecks;
+        Index[] storage calldataChecks = calldataList[contractAddress][selector];
 
         require(
             calldataChecks.length > 0, "CalldataList: No calldata checks found"
         );
-        require(
-            value <= calldataList[contractAddress][selector].maxValue,
```

Comment:
shouldn't there be a cap on value that can be sent in a transaction. If some users want no max value cap, they can set `maxValue` to `type(uint256).max`